action.yml

```diff
@@ -77,11 +82,15 @@ outputs:
     description: "true if the Java SDK was regenerated"
   java_directory:
     description: "The directory the Java SDK was generated to"
+  php_regenerated:
+    description: "true if the PHP SDK was regenerated"
+  php_directory:
+    description: "The directory the PHP SDK was generated to"
   commit_hash:
     description: "The commit hash if the SDK was regenerated"
 runs:
   using: "docker"
-  image: "docker://speakeasysagar/sdk-generation-action:v8"
+  image: "docker://ghcr.io/speakeasy-api/sdk-generation-action:v9"
```

Comment:
@simplesagar going to move to building and publishing to github packages for the docker image so it can be all managed internally by this repo